### ISSUE TITLE
Infinite Recursion on Some Video

### ISSUE THREAD

**Issue by Dheatly23** [Anarios/return-youtube-dislike#846]
### Have you tried to find similar open issues?

- [X] Yes, this issue is not a duplicate

### Browser

Mozilla Firefox

### Browser Version

Firefox 107.0.1

### Extension or Userscript?

Extension

### Extension/Userscript Version

3.0.0.7

### Video link where you see the problem

https://www.youtube.com/watch?v=K3CzPnQ9G5I

### What happened?

When opening video, a massive amount of lag happened. Dislike bar flickers frequently. Really hard to scroll, click, or do anything. Only by disabling the extension the lag disappears.

### How to reproduce/recreate?

1. Open the video.
2. Open console.
3. Wait until recursion errors (may take a minute).

Example screenshot:
![screenshot](https://user-images.githubusercontent.com/71598333/206882527-10e4a57d-80d2-48a5-b50e-baf39044236e.png)

Another example on different video:
![screenshot 2](https://user-images.githubusercontent.com/71598333/206887677-e150cb63-e7e0-47f7-abab-b9a0efedb0a9.png)

NOTE: May be caused by interaction with some other extensions (not exactly known what or why). Current combination that cause problems:

<details>
<summary>Click to expand</summary>

My combination:
1. uBlock Origin
2. Decentraleyes
3. Dark Reader
4. Breakthrough Twitter Login Wall
5. Multithreaded Download Manager
6. Youtube-shorts block

From discussion below:
1. uBlock Origin
2. Sponsorblock
3. Youtube-shorts block
4. Youtube Nonstop
5. Bypass Paywalls
6. enhanced-h264ify

From #859
1. Zoom Page WE

</details>

### Will you be available for follow-up questions to help developers diagnose & fix the issue?

Yes

**Comment by Anarios** [comment]
Thanks for report, so far I can't reproduce it. Must be something specific about your configuration.

**Comment by Dheatly23** [comment]
Found it! After quickly opening console i found this recursion error. It gets so laggy it crashed my browser. Will try to upload better quality later.

![image](https://user-images.githubusercontent.com/71598333/206882527-10e4a57d-80d2-48a5-b50e-baf39044236e.png)

**Comment by hallo1126** [comment]
I can confirm this issue, I have the exact same problem, with the exact same error message, with the problem being gone when I disable this specific addon. I have no visual feedback on the dislike bar though, but severe input lag on doing anything on the web page, the video runs too but is also affected by the input lag (trying to pause/unpause/skip though the video).

**It happens exclusively with videos of very short length, about less than 2 minutes I would guess.**

I have the following addons installed:

uBlock Origin
Sponsorblock
Youtube-shorts block
Youtube Nonstop
Bypass Paywalls
enhanced-h264ify

As far as I can tell, disabling either individual ones of them does not work, only disabling Return Youtube Dislike stops the issue, so I am not sure there is any conflict.

**Comment by TheGoldenSun** [comment]
I hope Anarios is able to fix this issue, because I'm also facing the same thing.
  


I can confirm this issue, I have the exact same problem, with the exact same error message, with the problem being gone when I disable this specific addon. I have no visual feedback on the dislike bar though, but severe input lag on doing anything on the web page, the video runs too but is also affected by the input lag (trying to pause/unpause/skip though the video).

It happens exclusively with videos of very short length, about less than 2 minutes I would guess.

I have the following addons installed:

uBlock Origin
Sponsorblock
Youtube-shorts block
Youtube Nonstop
Bypass Paywalls
enhanced-h264ify

As far as I can tell, disabling either individual ones of them does not work, only disabling Return Youtube Dislike stops the issue, so I am not sure there is any conflict.

**Comment by hallo1126** [comment]
@TheGoldenSun What is your browser addon list?

**Comment by hallo1126** [comment]
@TheGoldenSun What is your browser addon list and hardware?

**Comment by Dheatly23** [comment]
@hallo1126 @TheGoldenSun If i may ask, is it the same video i linked in the issue or are there any other example? Maybe there is some pattern here.

**Comment by kokonguyen191** [comment]
I also managed to recreate this issue quite consistently in firefox incognito with only return youtube dislike enabled.

Video where I got the issue, though the issue persists on almost every video: https://www.youtube.com/watch?v=DBCIuVopaGA

Flickering dislike bar

![2023-01-22 02-43-15 (online-video-cutter com)](https://user-images.githubusercontent.com/23214709/213882759-5b0c10a6-1060-45f7-90ed-5a81993677b5.gif)

Infinite recursion

![image](https://user-images.githubusercontent.com/23214709/213883200-c160e87a-25f5-43e5-8802-5abcd147b554.png)

I noticed my browser lagging accompanied by the flickering dislike bar and the ellipsis button kept jumping outside so I suspected an infinite loop and some element failing to resize themselves. I get the issue almost 100% of the time when I put the browser on my vertical screen (1080x1920) but the issue goes away when I put the browser on my horizontal screen (1920x1080), even during a video, then the issue returns when I put the browser back on my vertical screen, all during the same video.

The root cause could be different since I see OP got the issue even on wide screen but my test setup above allows me to reproduce the issue very consistently.
